katana/build.gradle

```diff
@@ -0,0 +1,23 @@
+apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+
+android {
+    compileSdkVersion 27
+
+    defaultConfig {
+        applicationId "com.mototdd.katana"
+        minSdkVersion 23
```

Comment:
@exallium - Why 23? Because this is the minimum version for insta apps + android auto.